src/AstrometryFit.cc

```diff
@@ -237,17 +124,17 @@ void AstrometryFit::LSDerivatives1(const CcdImage &ccdImage, TripletList &tList,
     // refraction stuff
     Point refractionVector = ccdImage.getRefractionVector();
     // transformation from sky to TP
-    const Gtransfo *sky2TP = _astrometryModel->getSky2TP(ccdImage);
+    const Gtransfo *sky2TP = _astrometryModel.getSky2TP(ccdImage);
     // reserve matrices once for all measurements
     GtransfoLin dypdy;
-    // the shape of h (et al) is required this way in order to be able to
+    // the shape of H (et al) is required this way in order to be able to
```

Comment:
Yeah, I switched those and was afraid you'd complain about it.

That said, I plan to eventually rename `H` because it's an unfortunate choice of name for a matrix that is *not* the Hessian.